src/app/auth/shared/store/auth-state.model.ts

```diff
@@ -0,0 +1,7 @@
+import { User } from '../models/user.model';
+
+export interface AuthState {
+  user: User;
+  error: string;
+  isAuth: boolean;
+}
```

Comment:
no need to put store on shared folder inside auth, just `.. app/auth/store/...`.
The same for actions, reducers etc.